### ISSUE TITLE
Could the default training hyperparameters be documented?

### ISSUE THREAD

**Issue by thesfy** [ICSResearch/HUNet#10]
Some of the hyperparameters (learning rate, batch size, number of iterations, optimizer configs, etc.) are not fully described in the README.
It would help if recommended values were added, especially for different CS ratios.
